src/components/tabs/components/tab/index.tsx

```diff
@@ -18,4 +18,5 @@ const Tab = (props: TabProps): ReactElement => {
 }
 
 export type { TabProps }
+export { Tab }
 export default Tab
```

Comment:
Are both of these exports needed?

Reply:
I suppose not! 🔪 Removed the default ones ✅